wasm_test_site/src/lib.rs

```diff
@@ -0,0 +1,31 @@
+use wasm_bindgen::prelude::*;
+
+fn log(string: impl AsRef<str>) {
+    web_sys::console::log_1(&JsValue::from_str(string.as_ref()));
+}
+
+#[wasm_bindgen]
+pub fn greet() {
+    console_error_panic_hook::set_once();
+
+    log("Hello, wasm_test_site!");
+}
+
+#[wasm_bindgen]
+pub fn test_hydroflow() -> web_sys::js_sys::Promise {
+    console_error_panic_hook::set_once();
+
+    let mut df = hydroflow::hydroflow_syntax! {
+        // https://hydro.run/docs/hydroflow/quickstart/example_1_simplest
+        source_iter(0..10) -> for_each(|n| log(format!("Hello {}", n)));
+    };
```

Comment:
Is the idea that this should be replaced with the client code?

Reply:
Yup